### PR TITLE
fix bug with `test: None`

### PR DESCRIPTION
**Pull request by wolfv**
There was a bug in graph creation becuase the test was set to None.

This fixes that. 

I also adjusted the code to extract requirements from the (multiple) tests and that should work now.

Example output for `carma`:

Note: we use the rendered `compiler`. Not sure if that is going to be an issue later on?

```json
{
    "feedstock_name": "carma",
    "parsing_error": false,
    "branch": "main",
    "raw_meta_yaml": "context:\n  name: carma\n  version: 0.8.0\n\npackage:\n  name: ${{ name|lower }}\n  version: ${{ version }}\n\nsource:\n  url: https://github.com/RUrlus/carma/archive/v${{ version }}.tar.gz\n  sha256: a404687fb7c41880794cb215cb678bc59cdb0ecdf8baa4336103e7834ba6750a\n  patches:\n    - patch-include-dir.patch\n\nbuild:\n  number: 1\n\nrequirements:\n  build:\n    - ${{ compiler('c') }}\n    - ${{ stdlib('c') }}\n    - ${{ compiler('cxx') }}\n    - cmake\n    - make\n\ntests:\n  - script:\n      - if: unix\n        then:\n          - test -f ${PREFIX}/include/carma/carma\n  - package_contents:\n      include:\n        - carma/carma\n\nabout:\n  homepage: https://github.com/RUrlus/carma\n  summary: A C++ header only library providing conversions between Numpy arrays and Armadillo matrices.\n  description: |\n    CARMA provides fast bidirectional conversions between Numpy arrays and\n    Armadillo matrices, vectors and cubes, much like RcppArmadillo does for R\n    and Armadillo.\n  license: Apache-2.0\n  license_file: LICENSE\n  documentation: https://carma.readthedocs.io\n  repository: https://github.com/RUrlus/carma\n\nextra:\n  recipe-maintainers:\n    - zklaus\n    - timkpaine\n    - wolfv\n",
    "conda-forge.yml": {
        "build_platform": {
            "osx_arm64": "osx_64"
        },
        "github": {
            "branch_name": "main",
            "tooling_branch_name": "main"
        },
        "conda_build": {
            "error_overlinking": true
        },
        "conda_forge_output_validation": true,
        "conda_build_tool": "rattler-build"
    },
    "platforms": [
        "linux_64",
        "osx_64",
        "osx_arm64",
        "win_64"
    ],
    "meta_yaml": {
        "about": {
            "description": "CARMA provides fast bidirectional conversions between Numpy arrays and\nArmadillo matrices, vectors and cubes, much like RcppArmadillo does for R\nand Armadillo.",
            "dev_url": "https://github.com/RUrlus/carma",
            "doc_url": "https://carma.readthedocs.io/",
            "home": "https://github.com/RUrlus/carma",
            "license": "Apache-2.0",
            "license_family": "Apache-2.0",
            "license_file": "LICENSE",
            "summary": "A C++ header only library providing conversions between Numpy arrays and Armadillo matrices."
        },
        "build": {
            "number": "1"
        },
        "package": {
            "name": "carma",
            "version": "0.8.0"
        },
        "requirements": {
            "build": [
                "gcc_osx-arm64 13.*",
                "sysroot_osx-arm64 2.17.*",
                "gxx_osx-arm64 13.*",
                "cmake",
                "make",
                "clang_osx-arm64 17.*",
                "macosx_deployment_target_osx-arm64 10.13.*",
                "clangxx_osx-arm64 17.*",
                "macosx_deployment_target_osx-arm64 11.0.*",
                "vs2019_osx-arm64",
                "vs_osx-arm64"
            ]
        },
        "source": {
            "patches": [
                "patch-include-dir.patch"
            ],
            "sha256": "a404687fb7c41880794cb215cb678bc59cdb0ecdf8baa4336103e7834ba6750a",
            "url": "https://github.com/RUrlus/carma/archive/v0.8.0.tar.gz"
        },
        "outputs": [
            {
                "name": "carma",
                "requirements": {
                    "build": [
                        "gcc_osx-arm64 13.*",
                        "sysroot_osx-arm64 2.17.*",
                        "gxx_osx-arm64 13.*",
                        "cmake",
                        "make"
                    ],
                    "host": [],
                    "run": []
                },
                "build": null,
                "tests": [
                    {
                        "script": "test -f ${PREFIX}/include/carma/carma"
                    },
                    {
                        "package_contents": {
                            "include": [
                                "carma/carma"
                            ]
                        }
                    }
                ]
            },
            {
                "name": "carma",
                "requirements": {
                    "build": [
                        "clang_osx-arm64 17.*",
                        "macosx_deployment_target_osx-arm64 10.13.*",
                        "clangxx_osx-arm64 17.*",
                        "cmake",
                        "make"
                    ],
                    "host": [],
                    "run": []
                },
                "build": null,
                "tests": [
                    {
                        "script": "test -f ${PREFIX}/include/carma/carma"
                    },
                    {
                        "package_contents": {
                            "include": [
                                "carma/carma"
                            ]
                        }
                    }
                ]
            },
            {
                "name": "carma",
                "requirements": {
                    "build": [
                        "clang_osx-arm64 17.*",
                        "macosx_deployment_target_osx-arm64 11.0.*",
                        "clangxx_osx-arm64 17.*",
                        "cmake",
                        "make"
                    ],
                    "host": [],
                    "run": []
                },
                "build": null,
                "tests": [
                    {
                        "script": "test -f ${PREFIX}/include/carma/carma"
                    },
                    {
                        "package_contents": {
                            "include": [
                                "carma/carma"
                            ]
                        }
                    }
                ]
            },
            {
                "name": "carma",
                "requirements": {
                    "build": [
                        "vs2019_osx-arm64",
                        "vs_osx-arm64",
                        "cmake",
                        "make"
                    ],
                    "host": [],
                    "run": []
                },
                "build": null,
                "tests": [
                    {
                        "script": "test -f ${PREFIX}/include/carma/carma"
                    },
                    {
                        "package_contents": {
                            "include": [
                                "carma/carma"
                            ]
                        }
                    }
                ]
            }
        ],
        "extra": {
            "recipe-maintainers": [
                "zklaus",
                "timkpaine",
                "wolfv"
            ]
        },
        "schema_version": 1
    },
    "linux_64_meta_yaml": {
        "about": {
            "description": "CARMA provides fast bidirectional conversions between Numpy arrays and\nArmadillo matrices, vectors and cubes, much like RcppArmadillo does for R\nand Armadillo.",
            "dev_url": "https://github.com/RUrlus/carma",
            "doc_url": "https://carma.readthedocs.io/",
            "home": "https://github.com/RUrlus/carma",
            "license": "Apache-2.0",
            "license_family": "Apache-2.0",
            "license_file": "LICENSE",
            "summary": "A C++ header only library providing conversions between Numpy arrays and Armadillo matrices."
        },
        "build": {
            "number": "1"
        },
        "package": {
            "name": "carma",
            "version": "0.8.0"
        },
        "requirements": {
            "build": [
                "gcc_osx-arm64 13.*",
                "sysroot_osx-arm64 2.17.*",
                "gxx_osx-arm64 13.*",
                "cmake",
                "make"
            ]
        },
        "source": {
            "patches": [
                "patch-include-dir.patch"
            ],
            "sha256": "a404687fb7c41880794cb215cb678bc59cdb0ecdf8baa4336103e7834ba6750a",
            "url": "https://github.com/RUrlus/carma/archive/v0.8.0.tar.gz"
        },
        "outputs": [
            {
                "name": "carma",
                "requirements": {
                    "build": [
                        "gcc_osx-arm64 13.*",
                        "sysroot_osx-arm64 2.17.*",
                        "gxx_osx-arm64 13.*",
                        "cmake",
                        "make"
                    ],
                    "host": [],
                    "run": []
                },
                "build": null,
                "tests": [
                    {
                        "script": "test -f ${PREFIX}/include/carma/carma"
                    },
                    {
                        "package_contents": {
                            "include": [
                                "carma/carma"
                            ]
                        }
                    }
                ]
            }
        ],
        "extra": {
            "recipe-maintainers": [
                "zklaus",
                "timkpaine",
                "wolfv"
            ]
        },
        "schema_version": 1
    },
    "linux_64_requirements": {
        "build": [
            "sysroot_osx-arm64",
            "cmake",
            "gxx_osx-arm64",
            "make",
            "gcc_osx-arm64"
        ],
        "host": [],
        "run": [],
        "test": []
    },
    "osx_64_meta_yaml": {
        "about": {
            "description": "CARMA provides fast bidirectional conversions between Numpy arrays and\nArmadillo matrices, vectors and cubes, much like RcppArmadillo does for R\nand Armadillo.",
            "dev_url": "https://github.com/RUrlus/carma",
            "doc_url": "https://carma.readthedocs.io/",
            "home": "https://github.com/RUrlus/carma",
            "license": "Apache-2.0",
            "license_family": "Apache-2.0",
            "license_file": "LICENSE",
            "summary": "A C++ header only library providing conversions between Numpy arrays and Armadillo matrices."
        },
        "build": {
            "number": "1"
        },
        "package": {
            "name": "carma",
            "version": "0.8.0"
        },
        "requirements": {
            "build": [
                "clang_osx-arm64 17.*",
                "macosx_deployment_target_osx-arm64 10.13.*",
                "clangxx_osx-arm64 17.*",
                "cmake",
                "make"
            ]
        },
        "source": {
            "patches": [
                "patch-include-dir.patch"
            ],
            "sha256": "a404687fb7c41880794cb215cb678bc59cdb0ecdf8baa4336103e7834ba6750a",
            "url": "https://github.com/RUrlus/carma/archive/v0.8.0.tar.gz"
        },
        "outputs": [
            {
                "name": "carma",
                "requirements": {
                    "build": [
                        "clang_osx-arm64 17.*",
                        "macosx_deployment_target_osx-arm64 10.13.*",
                        "clangxx_osx-arm64 17.*",
                        "cmake",
                        "make"
                    ],
                    "host": [],
                    "run": []
                },
                "build": null,
                "tests": [
                    {
                        "script": "test -f ${PREFIX}/include/carma/carma"
                    },
                    {
                        "package_contents": {
                            "include": [
                                "carma/carma"
                            ]
                        }
                    }
                ]
            }
        ],
        "extra": {
            "recipe-maintainers": [
                "zklaus",
                "timkpaine",
                "wolfv"
            ]
        },
        "schema_version": 1
    },
    "osx_64_requirements": {
        "build": [
            "cmake",
            "macosx_deployment_target_osx-arm64",
            "clang_osx-arm64",
            "make",
            "clangxx_osx-arm64"
        ],
        "host": [],
        "run": [],
        "test": []
    },
    "osx_arm64_meta_yaml": {
        "about": {
            "description": "CARMA provides fast bidirectional conversions between Numpy arrays and\nArmadillo matrices, vectors and cubes, much like RcppArmadillo does for R\nand Armadillo.",
            "dev_url": "https://github.com/RUrlus/carma",
            "doc_url": "https://carma.readthedocs.io/",
            "home": "https://github.com/RUrlus/carma",
            "license": "Apache-2.0",
            "license_family": "Apache-2.0",
            "license_file": "LICENSE",
            "summary": "A C++ header only library providing conversions between Numpy arrays and Armadillo matrices."
        },
        "build": {
            "number": "1"
        },
        "package": {
            "name": "carma",
            "version": "0.8.0"
        },
        "requirements": {
            "build": [
                "clang_osx-arm64 17.*",
                "macosx_deployment_target_osx-arm64 11.0.*",
                "clangxx_osx-arm64 17.*",
                "cmake",
                "make"
            ]
        },
        "source": {
            "patches": [
                "patch-include-dir.patch"
            ],
            "sha256": "a404687fb7c41880794cb215cb678bc59cdb0ecdf8baa4336103e7834ba6750a",
            "url": "https://github.com/RUrlus/carma/archive/v0.8.0.tar.gz"
        },
        "outputs": [
            {
                "name": "carma",
                "requirements": {
                    "build": [
                        "clang_osx-arm64 17.*",
                        "macosx_deployment_target_osx-arm64 11.0.*",
                        "clangxx_osx-arm64 17.*",
                        "cmake",
                        "make"
                    ],
                    "host": [],
                    "run": []
                },
                "build": null,
                "tests": [
                    {
                        "script": "test -f ${PREFIX}/include/carma/carma"
                    },
                    {
                        "package_contents": {
                            "include": [
                                "carma/carma"
                            ]
                        }
                    }
                ]
            }
        ],
        "extra": {
            "recipe-maintainers": [
                "zklaus",
                "timkpaine",
                "wolfv"
            ]
        },
        "schema_version": 1
    },
    "osx_arm64_requirements": {
        "build": [
            "cmake",
            "macosx_deployment_target_osx-arm64",
            "clang_osx-arm64",
            "make",
            "clangxx_osx-arm64"
        ],
        "host": [],
        "run": [],
        "test": []
    },
    "win_64_meta_yaml": {
        "about": {
            "description": "CARMA provides fast bidirectional conversions between Numpy arrays and\nArmadillo matrices, vectors and cubes, much like RcppArmadillo does for R\nand Armadillo.",
            "dev_url": "https://github.com/RUrlus/carma",
            "doc_url": "https://carma.readthedocs.io/",
            "home": "https://github.com/RUrlus/carma",
            "license": "Apache-2.0",
            "license_family": "Apache-2.0",
            "license_file": "LICENSE",
            "summary": "A C++ header only library providing conversions between Numpy arrays and Armadillo matrices."
        },
        "build": {
            "number": "1"
        },
        "package": {
            "name": "carma",
            "version": "0.8.0"
        },
        "requirements": {
            "build": [
                "vs2019_osx-arm64",
                "vs_osx-arm64",
                "cmake",
                "make"
            ]
        },
        "source": {
            "patches": [
                "patch-include-dir.patch"
            ],
            "sha256": "a404687fb7c41880794cb215cb678bc59cdb0ecdf8baa4336103e7834ba6750a",
            "url": "https://github.com/RUrlus/carma/archive/v0.8.0.tar.gz"
        },
        "outputs": [
            {
                "name": "carma",
                "requirements": {
                    "build": [
                        "vs2019_osx-arm64",
                        "vs_osx-arm64",
                        "cmake",
                        "make"
                    ],
                    "host": [],
                    "run": []
                },
                "build": null,
                "tests": [
                    {
                        "script": "test -f ${PREFIX}/include/carma/carma"
                    },
                    {
                        "package_contents": {
                            "include": [
                                "carma/carma"
                            ]
                        }
                    }
                ]
            }
        ],
        "extra": {
            "recipe-maintainers": [
                "zklaus",
                "timkpaine",
                "wolfv"
            ]
        },
        "schema_version": 1
    },
    "win_64_requirements": {
        "build": [
            "cmake",
            "vs_osx-arm64",
            "make",
            "vs2019_osx-arm64"
        ],
        "host": [],
        "run": [],
        "test": []
    },
    "total_requirements": {
        "build": [
            "macosx_deployment_target_osx-arm64 11.0.*",
            "clang_osx-arm64 17.*",
            "cmake",
            "gcc_osx-arm64 13.*",
            "vs_osx-arm64",
            "make",
            "clangxx_osx-arm64 17.*",
            "vs2019_osx-arm64",
            "sysroot_osx-arm64 2.17.*",
            "gxx_osx-arm64 13.*",
            "macosx_deployment_target_osx-arm64 10.13.*"
        ],
        "host": [],
        "run": [],
        "test": []
    },
    "requirements": {
        "build": [
            "sysroot_osx-arm64",
            "cmake",
            "macosx_deployment_target_osx-arm64",
            "clang_osx-arm64",
            "gxx_osx-arm64",
            "vs_osx-arm64",
            "make",
            "vs2019_osx-arm64",
            "clangxx_osx-arm64",
            "gcc_osx-arm64"
        ],
        "host": [],
        "run": [],
        "test": []
    },
    "strong_exports": false,
    "outputs_names": [
        "carma"
    ],
    "req": [
        "sysroot_osx-arm64",
        "cmake",
        "macosx_deployment_target_osx-arm64",
        "clang_osx-arm64",
        "gxx_osx-arm64",
        "vs_osx-arm64",
        "make",
        "vs2019_osx-arm64",
        "clangxx_osx-arm64",
        "gcc_osx-arm64"
    ],
    "name": "carma",
    "version": "0.8.0",
    "url": "https://github.com/RUrlus/carma/archive/v0.8.0.tar.gz",
    "hash_type": "sha256"
}
```

### CODE REVIEW

**Comment by beckermr** [comment]
Good spot on the compiler thing. Yes we need to use the stubs.

**Comment by wolfv** [comment]
Hmm, OK, I can rewrite the recipe to convert all ${{ compiler('c') }} to `c_compiler_stub` :)

Should we still merge thsi change beforehand or do you want this first, in the same PR?

**Comment by beckermr** [comment]
Same pr is fine.

File: conda_forge_tick/feedstock_parser.py [hunk 1]
```diff
@@ -154,9 +153,23 @@ def _extract_requirements(meta_yaml, outputs_to_keep=None):
             requirements_dict[section].update(
                 list(as_iterable(req.get(section, []) or [])),
             )
+
         test: "TestTypedDict" = block.get("test", {})
```
Comment (beckermr) [inline conda_forge_tick/feedstock_parser.py:157]:
```suggestion
        test: "TestTypedDict" = block.get("test", {}) or {}
```

**Comment by wolfv** [comment]
@beckermr after these changes, the rendered JSON looks like this. Please lmk what you think :)

```json
{
    "feedstock_name": "carma",
    "parsing_error": false,
    "branch": "main",
    "raw_meta_yaml": "context:\n  name: carma\n  version: 0.8.0\n\npackage:\n  name: ${{ name|lower }}\n  version: ${{ version }}\n\nsource:\n  url: https://github.com/RUrlus/carma/archive/v${{ version }}.tar.gz\n  sha256: a404687fb7c41880794cb215cb678bc59cdb0ecdf8baa4336103e7834ba6750a\n  patches:\n    - patch-include-dir.patch\n\nbuild:\n  number: 1\n\nrequirements:\n  build:\n    - ${{ compiler('c') }}\n    - ${{ stdlib('c') }}\n    - ${{ compiler('cxx') }}\n    - cmake\n    - make\n\ntests:\n  - script:\n      - if: unix\n        then:\n          - test -f ${PREFIX}/include/carma/carma\n  - package_contents:\n      include:\n        - carma/carma\n\nabout:\n  homepage: https://github.com/RUrlus/carma\n  summary: A C++ header only library providing conversions between Numpy arrays and Armadillo matrices.\n  description: |\n    CARMA provides fast bidirectional conversions between Numpy arrays and\n    Armadillo matrices, vectors and cubes, much like RcppArmadillo does for R\n    and Armadillo.\n  license: Apache-2.0\n  license_file: LICENSE\n  documentation: https://carma.readthedocs.io\n  repository: https://github.com/RUrlus/carma\n\nextra:\n  recipe-maintainers:\n    - zklaus\n    - timkpaine\n    - wolfv\n",
    "conda-forge.yml": {
        "build_platform": {
            "osx_arm64": "osx_64"
        },
        "github": {
            "branch_name": "main",
            "tooling_branch_name": "main"
        },
        "conda_build": {
            "error_overlinking": true
        },
        "conda_forge_output_validation": true,
        "conda_build_tool": "rattler-build"
    },
    "platforms": [
        "linux_64",
        "osx_64",
        "osx_arm64",
        "win_64"
    ],
    "meta_yaml": {
        "about": {
            "description": "CARMA provides fast bidirectional conversions between Numpy arrays and\nArmadillo matrices, vectors and cubes, much like RcppArmadillo does for R\nand Armadillo.",
            "dev_url": "https://github.com/RUrlus/carma",
            "doc_url": "https://carma.readthedocs.io/",
            "home": "https://github.com/RUrlus/carma",
            "license": "Apache-2.0",
            "license_family": "Apache-2.0",
            "license_file": "LICENSE",
            "summary": "A C++ header only library providing conversions between Numpy arrays and Armadillo matrices."
        },
        "build": {
            "number": "1"
        },
        "package": {
            "name": "carma",
            "version": "0.8.0"
        },
        "requirements": {
            "build": [
                "c_compiler_stub",
                "c_stdlib_stub",
                "cxx_compiler_stub",
                "cmake",
                "make"
            ]
        },
        "source": {
            "patches": [
                "patch-include-dir.patch"
            ],
            "sha256": "a404687fb7c41880794cb215cb678bc59cdb0ecdf8baa4336103e7834ba6750a",
            "url": "https://github.com/RUrlus/carma/archive/v0.8.0.tar.gz"
        },
        "outputs": [
            {
                "name": "carma",
                "requirements": {
                    "build": [
                        "c_compiler_stub",
                        "c_stdlib_stub",
                        "cxx_compiler_stub",
                        "cmake",
                        "make"
                    ],
                    "host": [],
                    "run": []
                },
                "build": null,
                "tests": [
                    {
                        "script": "test -f ${PREFIX}/include/carma/carma"
                    },
                    {
                        "package_contents": {
                            "include": [
                                "carma/carma"
                            ]
                        }
                    }
                ]
            },
            {
                "name": "carma",
                "requirements": {
                    "build": [
                        "c_compiler_stub",
                        "c_stdlib_stub",
                        "cxx_compiler_stub",
                        "cmake",
                        "make"
                    ],
                    "host": [],
                    "run": []
                },
                "build": null,
                "tests": [
                    {
                        "script": "test -f ${PREFIX}/include/carma/carma"
                    },
                    {
                        "package_contents": {
                            "include": [
                                "carma/carma"
                            ]
                        }
                    }
                ]
            },
            {
                "name": "carma",
                "requirements": {
                    "build": [
                        "c_compiler_stub",
                        "c_stdlib_stub",
                        "cxx_compiler_stub",
                        "cmake",
                        "make"
                    ],
                    "host": [],
                    "run": []
                },
                "build": null,
                "tests": [
                    {
                        "script": "test -f ${PREFIX}/include/carma/carma"
                    },
                    {
                        "package_contents": {
                            "include": [
                                "carma/carma"
                            ]
                        }
                    }
                ]
            },
            {
                "name": "carma",
                "requirements": {
                    "build": [
                        "c_compiler_stub",
                        "c_stdlib_stub",
                        "cxx_compiler_stub",
                        "cmake",
                        "make"
                    ],
                    "host": [],
                    "run": []
                },
                "build": null,
                "tests": [
                    {
                        "script": "test -f ${PREFIX}/include/carma/carma"
                    },
                    {
                        "package_contents": {
                            "include": [
                                "carma/carma"
                            ]
                        }
                    }
                ]
            }
        ],
        "extra": {
            "recipe-maintainers": [
                "zklaus",
                "timkpaine",
                "wolfv"
            ]
        },
        "schema_version": 1
    },
    "linux_64_meta_yaml": {
        "about": {
            "description": "CARMA provides fast bidirectional conversions between Numpy arrays and\nArmadillo matrices, vectors and cubes, much like RcppArmadillo does for R\nand Armadillo.",
            "dev_url": "https://github.com/RUrlus/carma",
            "doc_url": "https://carma.readthedocs.io/",
            "home": "https://github.com/RUrlus/carma",
            "license": "Apache-2.0",
            "license_family": "Apache-2.0",
            "license_file": "LICENSE",
            "summary": "A C++ header only library providing conversions between Numpy arrays and Armadillo matrices."
        },
        "build": {
            "number": "1"
        },
        "package": {
            "name": "carma",
            "version": "0.8.0"
        },
        "requirements": {
            "build": [
                "c_compiler_stub",
                "c_stdlib_stub",
                "cxx_compiler_stub",
                "cmake",
                "make"
            ]
        },
        "source": {
            "patches": [
                "patch-include-dir.patch"
            ],
            "sha256": "a404687fb7c41880794cb215cb678bc59cdb0ecdf8baa4336103e7834ba6750a",
            "url": "https://github.com/RUrlus/carma/archive/v0.8.0.tar.gz"
        },
        "outputs": [
            {
                "name": "carma",
                "requirements": {
                    "build": [
                        "c_compiler_stub",
                        "c_stdlib_stub",
                        "cxx_compiler_stub",
                        "cmake",
                        "make"
                    ],
                    "host": [],
                    "run": []
                },
                "build": null,
                "tests": [
                    {
                        "script": "test -f ${PREFIX}/include/carma/carma"
                    },
                    {
                        "package_contents": {
                            "include": [
                                "carma/carma"
                            ]
                        }
                    }
                ]
            }
        ],
        "extra": {
            "recipe-maintainers": [
                "zklaus",
                "timkpaine",
                "wolfv"
            ]
        },
        "schema_version": 1
    },
    "linux_64_requirements": {
        "build": [
            "c_compiler_stub",
            "cmake",
            "make",
            "cxx_compiler_stub",
            "c_stdlib_stub"
        ],
        "host": [],
        "run": [],
        "test": []
    },
    "osx_64_meta_yaml": {
        "about": {
            "description": "CARMA provides fast bidirectional conversions between Numpy arrays and\nArmadillo matrices, vectors and cubes, much like RcppArmadillo does for R\nand Armadillo.",
            "dev_url": "https://github.com/RUrlus/carma",
            "doc_url": "https://carma.readthedocs.io/",
            "home": "https://github.com/RUrlus/carma",
            "license": "Apache-2.0",
            "license_family": "Apache-2.0",
            "license_file": "LICENSE",
            "summary": "A C++ header only library providing conversions between Numpy arrays and Armadillo matrices."
        },
        "build": {
            "number": "1"
        },
        "package": {
            "name": "carma",
            "version": "0.8.0"
        },
        "requirements": {
            "build": [
                "c_compiler_stub",
                "c_stdlib_stub",
                "cxx_compiler_stub",
                "cmake",
                "make"
            ]
        },
        "source": {
            "patches": [
                "patch-include-dir.patch"
            ],
            "sha256": "a404687fb7c41880794cb215cb678bc59cdb0ecdf8baa4336103e7834ba6750a",
            "url": "https://github.com/RUrlus/carma/archive/v0.8.0.tar.gz"
        },
        "outputs": [
            {
                "name": "carma",
                "requirements": {
                    "build": [
                        "c_compiler_stub",
                        "c_stdlib_stub",
                        "cxx_compiler_stub",
                        "cmake",
                        "make"
                    ],
                    "host": [],
                    "run": []
                },
                "build": null,
                "tests": [
                    {
                        "script": "test -f ${PREFIX}/include/carma/carma"
                    },
                    {
                        "package_contents": {
                            "include": [
                                "carma/carma"
                            ]
                        }
                    }
                ]
            }
        ],
        "extra": {
            "recipe-maintainers": [
                "zklaus",
                "timkpaine",
                "wolfv"
            ]
        },
        "schema_version": 1
    },
    "osx_64_requirements": {
        "build": [
            "c_compiler_stub",
            "cmake",
            "make",
            "cxx_compiler_stub",
            "c_stdlib_stub"
        ],
        "host": [],
        "run": [],
        "test": []
    },
    "osx_arm64_meta_yaml": {
        "about": {
            "description": "CARMA provides fast bidirectional conversions between Numpy arrays and\nArmadillo matrices, vectors and cubes, much like RcppArmadillo does for R\nand Armadillo.",
            "dev_url": "https://github.com/RUrlus/carma",
            "doc_url": "https://carma.readthedocs.io/",
            "home": "https://github.com/RUrlus/carma",
            "license": "Apache-2.0",
            "license_family": "Apache-2.0",
            "license_file": "LICENSE",
            "summary": "A C++ header only library providing conversions between Numpy arrays and Armadillo matrices."
        },
        "build": {
            "number": "1"
        },
        "package": {
            "name": "carma",
            "version": "0.8.0"
        },
        "requirements": {
            "build": [
                "c_compiler_stub",
                "c_stdlib_stub",
                "cxx_compiler_stub",
                "cmake",
                "make"
            ]
        },
        "source": {
            "patches": [
                "patch-include-dir.patch"
            ],
            "sha256": "a404687fb7c41880794cb215cb678bc59cdb0ecdf8baa4336103e7834ba6750a",
            "url": "https://github.com/RUrlus/carma/archive/v0.8.0.tar.gz"
        },
        "outputs": [
            {
                "name": "carma",
                "requirements": {
                    "build": [
                        "c_compiler_stub",
                        "c_stdlib_stub",
                        "cxx_compiler_stub",
                        "cmake",
                        "make"
                    ],
                    "host": [],
                    "run": []
                },
                "build": null,
                "tests": [
                    {
                        "script": "test -f ${PREFIX}/include/carma/carma"
                    },
                    {
                        "package_contents": {
                            "include": [
                                "carma/carma"
                            ]
                        }
                    }
                ]
            }
        ],
        "extra": {
            "recipe-maintainers": [
                "zklaus",
                "timkpaine",
                "wolfv"
            ]
        },
        "schema_version": 1
    },
    "osx_arm64_requirements": {
        "build": [
            "c_compiler_stub",
            "cmake",
            "make",
            "cxx_compiler_stub",
            "c_stdlib_stub"
        ],
        "host": [],
        "run": [],
        "test": []
    },
    "win_64_meta_yaml": {
        "about": {
            "description": "CARMA provides fast bidirectional conversions between Numpy arrays and\nArmadillo matrices, vectors and cubes, much like RcppArmadillo does for R\nand Armadillo.",
            "dev_url": "https://github.com/RUrlus/carma",
            "doc_url": "https://carma.readthedocs.io/",
            "home": "https://github.com/RUrlus/carma",
            "license": "Apache-2.0",
            "license_family": "Apache-2.0",
            "license_file": "LICENSE",
            "summary": "A C++ header only library providing conversions between Numpy arrays and Armadillo matrices."
        },
        "build": {
            "number": "1"
        },
        "package": {
            "name": "carma",
            "version": "0.8.0"
        },
        "requirements": {
            "build": [
                "c_compiler_stub",
                "c_stdlib_stub",
                "cxx_compiler_stub",
                "cmake",
                "make"
            ]
        },
        "source": {
            "patches": [
                "patch-include-dir.patch"
            ],
            "sha256": "a404687fb7c41880794cb215cb678bc59cdb0ecdf8baa4336103e7834ba6750a",
            "url": "https://github.com/RUrlus/carma/archive/v0.8.0.tar.gz"
        },
        "outputs": [
            {
                "name": "carma",
                "requirements": {
                    "build": [
                        "c_compiler_stub",
                        "c_stdlib_stub",
                        "cxx_compiler_stub",
                        "cmake",
                        "make"
                    ],
                    "host": [],
                    "run": []
                },
                "build": null,
                "tests": [
                    {
                        "script": "test -f ${PREFIX}/include/carma/carma"
                    },
                    {
                        "package_contents": {
                            "include": [
                                "carma/carma"
                            ]
                        }
                    }
                ]
            }
        ],
        "extra": {
            "recipe-maintainers": [
                "zklaus",
                "timkpaine",
                "wolfv"
            ]
        },
        "schema_version": 1
    },
    "win_64_requirements": {
        "build": [
            "c_compiler_stub",
            "cmake",
            "make",
            "cxx_compiler_stub",
            "c_stdlib_stub"
        ],
        "host": [],
        "run": [],
        "test": []
    },
    "total_requirements": {
        "build": [
            "c_compiler_stub",
            "cmake",
            "make",
            "cxx_compiler_stub",
            "c_stdlib_stub"
        ],
        "host": [],
        "run": [],
        "test": []
    },
    "requirements": {
        "build": [
            "c_compiler_stub",
            "cmake",
            "make",
            "cxx_compiler_stub",
            "c_stdlib_stub"
        ],
        "host": [],
        "run": [],
        "test": []
    },
    "strong_exports": false,
    "outputs_names": [
        "carma"
    ],
    "req": [
        "c_compiler_stub",
        "cmake",
        "make",
        "cxx_compiler_stub",
        "c_stdlib_stub"
    ],
    "name": "carma",
    "version": "0.8.0",
    "url": "https://github.com/RUrlus/carma/archive/v0.8.0.tar.gz",
    "hash_type": "sha256"
}
```

**Pull request merged**